.buildkite/pipeline.yml

```diff
@@ -48,7 +51,8 @@ steps:
   - label: "cargo test nightly not integration-tests*"
     command: |
       source ~/.cargo/env && set -eux
-      cargo test --workspace --features nightly,test_features,mock_node -p '*' --exclude 'integration-tests*'
+      cargo install cargo-nextest
+      cargo nextest run --workspace --features nightly,test_features,mock_node -p '*' --exclude 'integration-tests*'
```

Comment:
Nextest does not run doc tests so we need another stage that runs `cargo test --doc`.

Reply:
done